code/game/objects/effects/mines.dm

```diff
@@ -28,11 +33,10 @@
 	s.set_up(3, 1, src)
 	s.start()
 	mineEffect(victim)
+	triggered = TRUE

```

Comment:
There is no wait between the trigger setting and the `qdel`, and signals are async so they can't sleep.
What race condition are you trying to handle here?

Reply:
The issue that caused this PR was multiple shrapnel mines going off next to each other would trigger their neighbors, who would shoot off shrapnel, who would immediately trigger their neighbors including the first mines before they went away, and if you had a 3x3 or so it'd get stuck in an infinite recursion and freeze the server. I removed flying bullets triggering landmines anyway, but this extra check is here to make sure the condition can't happen.

Reply:
Makes sense.